README.md

```diff
@@ -207,6 +207,9 @@ Minikube tools to be installed and available on your PATH.
    * `disable-linters` - Disable specific linters by name
    * `vscode-kubernetes.log-viewer.follow` - Set to true to follow logs by default in the log viewer.
    * `vscode-kubernetes.log-viewer.timestamp` - Set to true to show timestamps by default in the log viewer.
+   * `vscode-kubernetes.log-viewer.since` - The default value for the since field in the log viewer. Set to 0 for all logs.
```

Comment:
Might be worth mentioning examples of the format so people don't need to go look at it to find out?

Reply:
This is only for the number, there are 2 input fields that make up the duration.
`since-input` and `since-select`, from my understanding, the documentation suggests that the user could type it but it's inaccurate.

```js
function getSinceDuration() {
    const sinceType = document.getElementById('since-select').value;
    const sinceInput = document.getElementById('since-input').value;
    if (isNaN(sinceInput) || sinceInput <= 0 || sinceType.trim() === '') {
        return 0;
    }
    return `${sinceInput}${sinceType}`;
}
```

The string is actually built by using the number from `since-input` and appending the value from `since-select` (`h`, `m`, or `s`.)

Ofc it could be that these 2 input fields should be scrapped in favor of a single input field which is just a duration string like `5m`.

Please just let me know if you feel I've misunderstood anything.

Reply:
Gotcha - thanks for explaining.  In that case I think we do need to, as you discussed, save the value of the unit drop-down as well.  (Or, as you say, change the whole thing to free entry.)  E.g. if the user sets `since` to an hour and we save `1` but then the webview comes up with the drop-down being in minutes then that would be surprising...?